test/statsCases/define-plugin/webpack.config.js

```diff
@@ -18,5 +25,22 @@ module.exports = [
 				VALUE: "321"
 			})
 		]
+	},
+
+	{
+		mode: "production",
+		entry: "./index",
+		plugins: [
+			new webpack.DefinePlugin({
+				VALUE: webpack.DefinePlugin.runtimeValue(() => read("123.txt"), [
+					"./123.txt"
+				])
+			}),
+			new webpack.DefinePlugin({
+				VALUE: webpack.DefinePlugin.runtimeValue(() => read("321.txt"), [
+					"./321.txt"
+				])
+			})
+		]
```

Comment:
This test doesn't really tests the replacement and the dependencies. Better remove your changes from this test and add a test in `test/watchCases` instead. This test suite allow you to test watching and caching too.